src/bindings/gsettings/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 3.0)
 
 if (POLICY CMP0048)
-	cmake_policy(SET CMP0048 OLD) # avoid warning about resetting PROJECT_VERSION
+	cmake_policy(SET CMP0048 NEW) # avoid warning about resetting PROJECT_VERSION
 endif()
 
-project (elektrasettings)
+project (elektrasettings VERSION 0.1.1)
```

Comment:
Same as the above comment, this is about keeping this as possible out of Elektra build.